guides/release/components/triggering-changes-with-actions.md

```diff
@@ -103,34 +103,37 @@ in the child component we will implement the logic to confirm that the user want
 
 ```javascript {data-filename=app/components/button-with-confirmation.js}
 import Component from '@ember/component';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  actions: {
-    launchConfirmDialog() {
-      this.set('confirmShown', true);
-    },
+export default class ButtonWithConfirmation extends Component {
+  @action
+  launchConfirmDialog() {
+    this.set('confirmShown', true);
+  }
 
-    submitConfirm() {
-      // trigger action on parent component
-      this.set('confirmShown', false);
-    },
+  @action
+  submitConfirm() {
+    // trigger action on parent component
+    this.set('confirmShown', false);
+  }
 
-    cancelConfirm() {
-      this.set('confirmShown', false);
-    }
+  @action
+  cancelConfirm() {
+    this.set('confirmShown', false);
   }
-});
+}
 ```
 
 The component template will have a button and a div that shows the confirmation dialog
 based on the value of `confirmShown`.
 
 ```handlebars {data-filename=app/templates/components/button-with-confirmation.hbs}
-<button {{action "launchConfirmDialog"}}>{{this.text}}</button>
+<button {{action this.launchConfirmDialog}}>{{this.text}}</button>
+
 {{#if this.confirmShown}}
   <div class="confirm-dialog">
-    <button class="confirm-submit" {{action "submitConfirm"}}>OK</button>
-    <button class="confirm-cancel" {{action "cancelConfirm"}}>Cancel</button>
+    <button class="confirm-submit" {{action this.submitConfirm}}>OK</button>
```

Comment:
@locks @pzuraq should we be using the modifier version of `action` in the examples, since this won't bubble anyway and is less clear then `onclick={{this.submitConfirm}}`.

Reply:
I'm not sure about that. I think ideally if we used a modifier version, we would use something like the [on modifier](https://github.com/buschtoens/ember-on-modifier#readme), `{{action}}` has too many quirks and acts like jquery action handlers which are not ideal. Since that's not mainline, I would lean toward `onclick=` (cc @tomdale I believe he has some opinions here too)

Reply:
I converted to `onclick=...`. Let me know if I should change it to something else.